### PR TITLE
Add full website for Math Magicians Project

### PR DESCRIPTION
**Pull request by demix007**
## In this milestone of my Math Magicians project, I implemented the following:

- Created a website consisting of 3 pages: Home, Calculator, and Quote.
- Ensured that the general layout matched the provided wireframes provided in the project requirements.
-  Added my own styles to improve the look and feel of the website using react basic styling
-  Created a route for every page.
- Made sure that navigation links for all 3 routes are displayed on each page. 
- Used React Router to make routes work. 
- Made sure that there were no linter errors
- Used the correct Gitflow
- Documented the project in a professional way. 
- Followed Javascript best practices

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -24,33 +24,37 @@ const Calculator = () => {
...
+          <button type="button" className="button six" onClick={handleButtonClick}>6</button>
+          <button type="button" className="button minus orange" onClick={handleButtonClick}>-</button>
+          <button type="button" className="button one" onClick={handleButtonClick}>1</button>
+          <button type="button" className="button two" onClick={handleButtonClick}>2</button>
+          <button type="button" className="button three" onClick={handleButtonClick}>3</button>
+          <button type="button" className="button plus orange" onClick={handleButtonClick}>+</button>
+          <button type="button" className="button zero double-row" onClick={handleButtonClick}>0</button>
+          <button type="button" className="button dot" onClick={handleButtonClick}>.</button>
+          <button type="button" className="button equal orange" onClick={handleButtonClick}>=</button>
+        </div>
```
Comment (mckent05) [inline src/components/calculator.js:54]: - Please ensure that your calculator page is aligned the same way it is in the wireframe. Remember it is important that you try to replicate the wireframe design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
- The calculator should be aligned on the right side of the text. Please fix this 👍 
bdvbdhbvdbv

Wireframe:
ssnjnjn

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @demix007,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- Your project looks good 👍 👍 
- Your PR is professional 👏 
- All linter checks are passing correctly ✔️ ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by demix007** [comment]
## Hello @mckent05 👋🏼,

- Thank you for taking out time to review this pull request. The changes required for the wireframe have been made just as you suggested, thanks for pointing that out. 💯 I have resubmitted the PR for review.

**Review by andersonlebon — APPROVED** [review]
## Hi @demix007 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
- [x] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by demix007** [comment]
## Hello @andersonlebon 😎, 

Thank you for taking out time to review and approve this PR. 👍🏼

**Pull request merged**